### PR TITLE
Test fixes

### PR DESCRIPTION
**Pull request by it-ony**
# Problem

npm run test:unit was not executed as part of the github pipeline. PR for this is [here](https://github.com/onfido/onfido-sdk-ui/pull/1623).

Some tests were failing most times to missing function in mocks.  

# Solution

Added missing function + fixing mocking of http requests

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the TESTING_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [X] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by kopijunkie — APPROVED** [review]
LGTM

File: src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts [hunk 1]
```diff
@@ -47,105 +62,119 @@ describe('onfidoApi', () => {
...
+            invalidSnapshotData,
+            invalidSelfieData,
+            jwtToken,
+            url,
+            () => {
+              error('success should not be invoked')
+            },
+            (e) => {
+              // @ts-ignore
+              const ex: TypeError = e as TypeError
```
Comment (stefaniacardenas) [inline src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts:158]: Can we call this `error` or `err`? `ex` is not as readable

File: src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts [hunk 2]
```diff
@@ -47,105 +62,119 @@ describe('onfidoApi', () => {
...
+              } catch (e) {
+                console.error(e)
+              }
+            },
+            mockedTrackingCallback
+          )
+
+          xhrMock.onload && xhrMock.onload(new ProgressEvent('Type error'))
+
+          expect(xhrMock.send).not.toHaveBeenCalled()
```
Comment (stefaniacardenas) [inline src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts:176]: Shouldn't you add an additional expectation to check that the mockedOnError has been called?

File: src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts (hunk 1 above)
Comment (it-ony) [reply src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts:158]: Naming ... 

`error` is used already.
`ex` is for exception

Anyway, renamed it for err. Feel free in the future to rename it by yourself, to save both of us time.

File: src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts (hunk 2 above)
Comment (it-ony) [reply src/components/utils/__tests__/onfidoApi/sendMultiframeSelfie.test.ts:176]: Good spot.

**Review by stefaniacardenas — APPROVED** [review]

**Pull request merged**